client/src/varserver.c

```diff
@@ -696,21 +696,20 @@ int VARSERVER_Test( VARSERVER_HANDLE hVarServer )
     VarClient *pVarClient = ValidateHandle( hVarServer );
     if( pVarClient != NULL )
     {
-        for(i=0;i<100;i++)
+        for(i=0;i<1;i++)
```

Comment:
maybe ditch the for loop completely instead of setting loop count to 1

Reply:
done